### ISSUE TITLE
Orchid Linux Client Crashes on Ubuntu 20.04 because of netmask

### ISSUE THREAD

**Issue by therishidesai** [OrchidTechnologies/orchid#64]
**Describe the bug**
When running orchidcd-lnx_0.9.22 it fails with the following error:
```
route: netmask doesn't match route address
Usage: inet_route [-vF] del {-host|-net} Target[/prefix] [gw Gw] [metric M] [[dev] If]
       inet_route [-vF] add {-host|-net} Target[/prefix] [gw Gw] [metric M]
                              [netmask N] [mss Mss] [window W] [irtt I]
                              [mod] [dyn] [reinstate] [[dev] If]
       inet_route [-vF] add {-host|-net} Target[/prefix] [metric M] reject
       inet_route [-FC] flush      NOT supported
[source/main.cpp:95] system(route -n add -net 1/8 dev tun0)
terminate called without an active exception
[1]    212615 abort      sudo ./orchidcd-lnx_0.9.22
```
I am running on Ubuntu 20.04

**To Reproduce**
1. Download the orhcidcd-lnx_0.9.22 binary
2. Run the binary with sudo

**Device information:**
 - Device: Dell XPS 15
 - OS: Ubuntu 20.04
 - Application Version: 0.9.22

**Comment by saurik** [comment]
@apache8080 Can we get the output of ```ip a``` and ```ip route show```?

**Comment by apache8080** [comment]
output of `ip a`
```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: wlp59s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 90:78:41:5b:20:ac brd ff:ff:ff:ff:ff:ff
    inet 192.168.86.197/24 brd 192.168.86.255 scope global dynamic noprefixroute wlp59s0
       valid_lft 80268sec preferred_lft 80268sec
    inet6 fe80::9023:a17d:4e61:cd3b/64 scope link noprefixroute
       valid_lft forever preferred_lft forever
3: virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
    link/ether 52:54:00:01:eb:0c brd ff:ff:ff:ff:ff:ff
    inet 192.168.122.1/24 brd 192.168.122.255 scope global virbr0
       valid_lft forever preferred_lft forever
4: virbr0-nic: <BROADCAST,MULTICAST> mtu 1500 qdisc fq_codel master virbr0 state DOWN group default qlen 1000
    link/ether 52:54:00:01:eb:0c brd ff:ff:ff:ff:ff:ff
5: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default
    link/ether 02:42:e2:b9:70:35 brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
       valid_lft forever preferred_lft forever
 ```

output of `ip route show`:
```
default via 192.168.86.1 dev wlp59s0 proto dhcp metric 600
169.254.0.0/16 dev virbr0 scope link metric 1000 linkdown
172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1 linkdown
192.168.86.0/24 dev wlp59s0 proto kernel scope link src 192.168.86.197 metric 600
192.168.122.0/24 dev virbr0 proto kernel scope link src 192.168.122.1 linkdown
```

**Comment by saurik** [comment]
@therishidesai I'm sorry we didn't get back to you on this before :(, but we are pretty sure we figured out what was causing this and fixed it in 0d57599a6c250e4855676778a34b4d58e7c54adf (though, technically, this fix wasn't included in an official "release"--not that I _really_ recommend anyone relies on those...--until two days ago).
